### DISCUSSION TITLE
Neues Gerät hinzufügen: Servicepartner in Schritt 2

### DISCUSSION THREAD

**Discussion by moritzvolkmar** [Samedis-care/samedis-care-community#27] [Idea retired / denied]
Wenn man ein neues Gerät hinzufügt, wird aktuell bereits in Schritt 1 nach dem **Servicepartner** gefragt. Wenn der Datensatz gespeichert ist, findet man den **Servicepartner** unter _Service_. 

Beim Anlegen des Gerätes müsste der **Servicepartner** daher auch in Step 2.

**Comment by YvesR** [comment]
Stepper und Reiter sind nicht 1:1 und einen Service Partner unter Events anlegen ergibt zudem im Kontext nicht so viel Sinn.
